internal/pkg/model/mapper.go

```diff
@@ -163,15 +163,15 @@ type LocalSaveRecipe struct {
 
 // RemoteLoadRecipe - all items related to the object, when loading from Storage API.
 type RemoteLoadRecipe struct {
-	Manifest       ObjectManifest
+	ObjectManifest
 	ApiObject      Object // eg. Config, original version, API representation
 	InternalObject Object // eg. Config, modified version, internal representation
 }
 
 // RemoteSaveRecipe - all items related to the object, when saving to Storage API.
 type RemoteSaveRecipe struct {
-	ChangedFields  ChangedFields
-	Manifest       ObjectManifest
+	ChangedFields ChangedFields
+	ObjectManifest
```

Comment:
`ObjectManifest` field je uz embedded, je to zjednotene s LocaRecipe:
https://github.com/keboola/keboola-as-code/blob/750c3138daec0d9b6b4cc00f89db10d3eca4d272/internal/pkg/model/mapper.go#L155-L162

A daju sa nad `recipe` volat metody  z `ObjectManfiest`, napr. `recipe.Desc`.